frontend/src/scenes/sceneLogic.ts

```diff
@@ -303,7 +303,7 @@ export const sceneLogic = kea<sceneLogicType<LoadedScene, Params, Scene, SceneCo
             const { user } = userLogic.values
             const { preflight } = preflightLogic.values
 
-            if (scene === Scene.Signup && preflight && !preflight.cloud && preflight.initiated) {
+            if (scene === Scene.Signup && preflight && !preflight.can_create_org) {
```

Comment:
Note this change, we now delegate to the backend on whether a user can create a new organization in this instance.